src/CLICommand/parameters_helper.ts

```diff
@@ -246,9 +246,13 @@ export class ParametersHelper {
 	 * @returns {string | undefined}
 	 * Returns the string value for the specific parameter; returns undefined if not set
 	 */
-	public getParameterValue(parameterName: ParameterName): string | undefined {
+	public getParameterValue<T = string>(
```

Comment:
This change is required to use the functionality of the `commander` API to declare an array parameter